code/modules/surgery/organs/external/wings.dm

```diff
@@ -50,15 +50,13 @@
 
 	fly.Remove(organ_owner)
 
-/obj/item/organ/external/wings/functional/on_life(delta_time, times_fired)
+/obj/item/organ/external/wings/functional/process(delta_time, times_fired)
```

Comment:
i said this on discord but this isnt the right way to fix this. you need to add wings to GLOB.organ_process_order so this on_life() actually processes